lib/widgets/drive.coffee

```diff
@@ -51,18 +50,34 @@ getDrives = ->
 # visuals.drive('Please select a drive').then (drive) ->
 # 	console.log(drive)
 ###
-module.exports = (message = 'Select a drive') ->
-	getDrives().then (drives) ->
 
-		if _.isEmpty(drives)
-			throw new Error('No available drives')
+module.exports = (message = 'Select a drive') ->
 
-		form.ask
-			type: 'list'
-			name: 'drive'
+	getDrives().then (drives) ->
+		options =
 			message: message
+			name: 'list'
 			choices: _.map drives, (drive) ->
 				return {
 					name: "#{drive.device} (#{drive.size}) - #{drive.description}"
 					value: drive.device
 				}
+
+		ui = new UI
+			input: process.stdin
+			zoutput: process.stdout
+		list = new InquirerList(options, ui.rl)
+		scanner = new DriveScanner(getDrives, interval: 5000, drives: drives)
+
+		scanner.on 'add', (drive) ->
+			list.opt.choices.push("#{drive.device} (#{drive.size}) - #{drive.description}")
+			list.render()
+
+		scanner.on 'remove', (drive) ->
+			list.opt.choices.choices = _.filter list.opt.choices.choices, (x) ->
+				return x.name != "#{drive.device} (#{drive.size}) - #{drive.description}"
```

Comment:
@jviotti this is an example of list.opt.choices.choices
```
   [ { name: '\\\\.\\PHYSICALDRIVE2 (7 GB) - TOSHIBA TransMemory USB Device',
       value: '\\\\.\\PHYSICALDRIVE2',
       disabled: undefined } ],
```
and this is one from drive
```
{ device: '\\\\.\\PHYSICALDRIVE2',
  description: 'TOSHIBA TransMemory USB Device',
  size: '7 GB',
  mountpoint: 'D:' }
```
so besides adding a check for the choices.value being equal to drive.device i can't think of another way